crates/router/src/connector/worldpay/response.rs

```diff
@@ -218,11 +221,11 @@ impl PaymentInstrumentCardIssuer {
 #[serde(rename_all = "camelCase")]
 pub struct PaymentInstrumentCardNumber {
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub bin: Option<String>,
+    pub bin: Option<Secret<String>>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub last4_digits: Option<String>,
+    pub last4_digits: Option<Secret<String>>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub dpan: Option<String>,
+    pub dpan: Option<Secret<String>>,
```

Comment:
These are not required to be masked, please take @ArjunKarthik 's view on this once